src/main/java/org/jenkinsci/plugins/mesos/JenkinsScheduler.java

```diff
@@ -219,9 +226,13 @@ public synchronized void terminateJenkinsSlave(String name) {
              return;
            }
         }
+
         LOGGER.warning("Asked to kill unknown mesos task " + taskId);
     }
 
+    // Since this task is now running, we should not start this task up again at a later point in time
```

Comment:
No, it's correct. This adds the task to the set so it won't run again later

On June 21, 2015 9:49:35 PM CDT, Vinod Kone <notifications@github.com> wrote:
>>          LOGGER.warning("Asked to kill unknown mesos task " +
>taskId);
>>      }
>>  
>> +    // Since this task is now running, we should not start this task
>up again at a later point in time
>
>s/now/not/ ?
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/jenkinsci/mesos-plugin/pull/124/files#r32902602

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.